modules/database/src/adapters/sequelize-adapter/index.ts

```diff
@@ -1,11 +1,21 @@
-import { Sequelize } from 'sequelize';
+import { QueryTypes, Sequelize } from 'sequelize';
 import { SequelizeSchema } from './SequelizeSchema';
 import { schemaConverter } from './SchemaConverter';
-import { ConduitSchema, GrpcError } from '@conduitplatform/grpc-sdk';
+import { ConduitModel, ConduitSchema, GrpcError } from '@conduitplatform/grpc-sdk';
 import { systemRequiredValidator } from '../utils/validateSchemas';
 import { DatabaseAdapter } from '../DatabaseAdapter';
-import { stitchSchema } from "../utils/extensions";
+import { stitchSchema } from '../utils/extensions';
 import { status } from '@grpc/grpc-js';
+import { SequelizeAuto, TableData } from 'sequelize-auto';
+import {
+  sqlSchemaConverter,
+  INITIAL_DB_SCHEMAS,
+} from '../../introspection/sequelize/utils';
+import { isNil } from 'lodash';
+import { isMatch } from 'lodash';
+import { MultiDocQuery } from '../../interfaces';
+
+const sqlSchemaName = process.env.SQL_SCHEMA ?? 'public';
```

Comment:
Consider specifying Sequelize's `options.searchPath` so that we don't need to explicitly specify this everywhere as that could potentially be a root cause for unexpected bugs down the line as soon as we forget about it.

[Link to Sequelize docs](https://sequelize.org/api/v6/class/src/model.js~model) (options.searchPath)

Reply:
We could add the searchPath option however it cannot be provided to the sequelize constructor, meaning that we would need to explicitly add it to every CRUD operation on the models themselves rather than the sequelize instance.